src/Codec/Xlsx/Parser/Stream.hs

```diff
@@ -141,6 +140,7 @@ data SheetState = MkSheetState
   , _ps_sheet_name     :: Text            -- ^ Current sheet name
   , _ps_cell_row_index :: Int             -- ^ Current row number
   , _ps_cell_col_index :: Int             -- ^ Current column number
+  , _ps_cell_style     :: Maybe Int
```

Comment:
As a layman _not_ familiar with the XLSX spec, it is surprising to me that the style of a cell is modelled with an optional integer.

Reply:
it's a reference, there are too many cells to individually format them all so they made another lookup table.